examples/ansible/roles/alp_reboot/README.md

```diff
@@ -0,0 +1,86 @@
+
+# Ansible Role: alp_reboot
+
+## Overview
+
+The `alp_reboot` role is designed to manage system reboots in an Ansible-controlled environment. It checks for the existence of a specific systemd service unit file, removes it if it does exist, and then recreates it based on a specific template. This role also manages the lingering settings for a specific user and initiates a system reboot when necessary.
+
+
+## Role Variables
+
+The main variable used in this role is `ansible_user`, which specifies the user account to enable lingering for and to execute tasks that require elevated privileges. Additionally, `playbook_path`, `ansible_playbook_basename`, and `reboot_mode` are also defined in this role.
+
+## Tasks Included in This Role
+
+Here's an overview of the tasks executed by this role:
+
+1. Enable lingering for the `ansible_user`.
+2. Check if the service unit file `/etc/systemd/system/myplaybook.service` exists. If it does, remove it.
+3. Create a systemd service unit file from the template `myplaybook.service.j2` only if it did not exist previously.
+4. Enable and start the `myplaybook` service, only if the service unit file did not previously exist.
+5. Reboot the system if the service unit file did not previously exist.
+6. Remove the `myplaybook.service` service file.
```

Comment:
Same goes for the `myplaybook` references here and throughout the rest of the PR.

Reply:
renamed myplaybook service to resume-ansible-playbook in the role templates,cleanup.yml and setup.yml, will update in the readme shortly.